src/app/app.component.css

```diff
@@ -35,6 +35,8 @@ body {
     --button-text2: #4285F4;
     --table-header: #545454;
     --scrollbar-color: rgba(84, 84, 84, 0.17);
+    --togglebutton-color: #f3d4d6;
+    --text-color: #ccc4c4;

```

Comment:
Use Global variables for test color, and button toggle color

Reply:
Resolved changes